### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by rainleander**
Signed-off-by: K Rain Leander <rainleander@users.noreply.github.com>

## Description

Contributing notes may be included in README.md or in a separate CONTRIBUTING.md. Expectations around licensing, copyright, and what to expect when contributing to this project should be covered. Tips on what linters and tests are run should be provided so that users can perform these locally. These may differ from project to project, even among common programming languages, based on the ecosystem of the project.

Fixes: https://github.com/tinkerbell/tinkerbell.org/issues/107

### CODE REVIEW

**Comment by mmlb** [comment]
@rainleander We are going to put this in .github repo?

**Comment by gianarb** [comment]
I think this is out of date! We have a CoC available here https://github.com/tinkerbell/.github/blob/master/CODE_OF_CONDUCT.md

**Comment by rainleander** [comment]
A code of conduct, a document around the cultural norms / acceptable behaviour, isn't the same as the contributing file which goes into details as to how to contribute to a project. Both are necessary.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Contributors
+
+Hello Contributors!  Thx for your interest! We're so glad you're here. 
+
+## Important Resources
+  - bugs: [https://github.com/tinkerbell/tinkerbell.org/issues](https://github.com/tinkerbell/tinkerbell.org/issues)
+  - feature requests: [https://github.com/tinkerbell/tinkerbell.org/issues](https://github.com/tinkerbell/tinkerbell.org/issues)
+
+### Code of Conduct
+Available via [https://github.com/tinkerbell/tinkerbell.org/blob/master/.github/CODE_OF_CONDUCT.md](https://github.com/tinkerbell/tinkerbell.org/blob/master/.github/CODE_OF_CONDUCT.md)
```
Comment (mmlb) [inline CONTRIBUTING.md:10]: Can you please update this to https://github.com/tinkerbell/.github/blob/master/CODE_OF_CONDUCT.md

**Review by mmlb — CHANGES_REQUESTED** [review]

**Comment by tstromberg** [comment]
Closed as obsolete.

**Pull request closed without merging**